docker-compose-prod.yml

```diff
@@ -0,0 +1,38 @@
+services:
+  app:
+    image: archivesspace/archivesspace:@TAG@
```

Comment:
@mark-cooper The @TAG@ token is set by ant: https://github.com/archivesspace/archivesspace/pull/3420/files#diff-bffc59192cdccaf509753556461ff111ed11b8611616b7ebf7bbfd0700c3a711R318

to the git tag as passed to it from the github workflow:  https://github.com/archivesspace/archivesspace/pull/3420/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R47

Which leaves this strange @TAG@ token in the compose file but serves our release workflow... I do not have a better idea for this...

Reply:
I think it looks fine. A possible alternative might be to use `latest` as the default in place of `@TAG@` so it is runnable locally if you want to try it out easily (i.e. with `docker compose -f docker-compose.prod.yml up -d` for testing). But that exposes a potential issue in the case that "latest" isn't properly replaced during the build workflow (although then there could be checks for that etc.).